File: 58 Length of Last Word/solution.py
class Solution:
    # @param {string} s
    # @return {integer}
    def lengthOfLastWord(self, s):
        if len(s)==0:
            return 0
        result = ""
        for i in range(len(s)):
            if s[-1-i]!=" ":
                result+=s[-1-i]
            elif len(result)!=0:
                break
        return len(result)
        

File: 58 Length of Last Word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        string result="";
        int leng = s.size();
        for (int i=leng-1;i>=0;i--){
            if (s[i]!=' ')
                result+=s[i];
            else if (result.size()!=0)
                break;
        }
        return result.size();
    }
};